lte/protos/session_manager.proto

```diff
@@ -348,6 +362,7 @@ message CreditUsage {
   UpdateType type = 5;
   bytes monitoring_key = 6;
   ServiceIdentifier service_identifier = 7;
+  RequestedUnits requested_units = 8;
```

Comment:
@uri200 Can you suggest a good place to place the 'RequestedUnits' here? I thought you added a similar thing but I don't see it in this base commit.

Reply:
Oh actually, is this the field you added?

Reply:
@uri200 , Pls. respond to Marie. And  can I resolve this comment?

Reply:
Yes, after reviewing the previous comment I see this is the field I added few days ago. 

So it should be left there